MovieTime/MovieTime/View/Movies/MTMoviesFeedViewController.swift

```diff
@@ -0,0 +1,231 @@
+//
+//  MTMoviesFeedViewController.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 08/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import UIKit
+
+class MTMoviesFeedViewController: MTViewController {
+
+    // MARK: Outlets
+    @IBOutlet weak var collectionView: UICollectionView!
```

Comment:
the white background from this page often blends with part of the movie tiles. What UI changes do you suggest to improve this?

Reply:
Using a dark border would solve the problem or just change the `shadowOffset` of the *CollectionViewCell* and the shadow will make an effect between the movie tile and the background